### ISSUE TITLE
How to have the `INamingConvention` passed into a custom value converter?

### ISSUE THREAD

**Issue by rabejens** [aaubry/YamlDotNet#816]
I wrote a custom value converter. I initialize it like this:

```csharp
var ser = new SerializerBuilder().WithNamingConvention(UnderscoredNamingConvention.Instance).WithTypeConverter(MyTypeConverter.Instance).Build();
```

How to get the naming convention into the `MyTypeConverter` without having to explicitly pass it e.g. via a constructor parameter?

**Comment by EdwardCooke** [comment]
I don't believe the naming convention is passed down in to the type converter and you'll need to pass it in through a constructor. Do you have example code of your `MyTypeConverter` class we could look at? But, I'm pretty sure it's not going to be available.

**Comment by EdwardCooke** [comment]
I haven't heard back in a couple of months. Do you still need help with this. Currently the naming convention isn't passed down into the type converter. We may be able to add something into it that would allow it to be passed in, but it would end up being a breaking change which we try to not do unless there is some dire reason we have to.
